src/karmen_backend/server/models/users_me.py

```diff
@@ -0,0 +1,116 @@
+import hashlib
+from datetime import datetime, timedelta
+import uuid as guid
+import bcrypt
+from server.database import (
+    users,
+    local_users,
+    organizations,
+    organization_roles,
+)
+from server.services.validators import is_email
+
+
+def create_inactive_user(email):
+    existing = users.get_by_email(email)
+    # activated users
+    if existing and existing["activated"] is not None:
```

Comment:
`is not None`? I would expect a boolean flag..

Reply:
`ativated` is a  `timestamp` column in databas. `None` for not activated users, timestamp for activated.